### PR TITLE
Go binding demo

### PR DESCRIPTION
**Pull request by ksyeo1010**
- added go binding and demo

### CODE REVIEW

File: binding/go/leopard_posix.go [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+		audio_path,
+		transcript);
+}
+
+typedef void (*pv_leopard_delete_func)(void *);
+
+void pv_leopard_delete_wrapper(void *f, void *object) {
+	return ((pv_leopard_delete_func) f)(object);
+}
+*/
```
Comment (ErisMik) [inline binding/go/leopard_posix.go:92]: Is this comment supposed to be here?

**Review by ErisMik — APPROVED** [review]

File: binding/go/leopard_posix.go (hunk 1 above)
Comment (ksyeo1010) [reply binding/go/leopard_posix.go:92]: yes, thats how go calls c funcs

Comment (ErisMik) [reply binding/go/leopard_posix.go:92]: Okay, I thought so but wanted to make sure.

**Comment by ErisMik** [comment]
LGTM

**Pull request merged**